app/ts/library/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { getAddress } from 'ethers';
+import { EthereumAddress } from '../types/ethereumTypes.js';
+import { serialize } from '../types/types.js';
+
+export function addressString(address: bigint): string {
+	return getAddress(serialize(EthereumAddress, address).toLowerCase())
```

Comment:
```suggestion
	return getAddress(serialize(EthereumAddress, address))
```

serialize produces lowercased string already, and even if it didn't, getAddress can handle non lower cased one too

Reply:
You sure the serialize does? I know `ethers`'s getAddress would error if passed a non-checksumed string that wasn't all lowercase. Not sure if thats recently been fixed.